datadog_checks_base/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG - datadog_checks_base
 
+## 31.0.0 / 2023-04-14
+
+* [Added] Update dependencies. See [#14357](https://github.com/DataDog/integrations-core/pull/14357).
+* [Added] Update redis to 4.5.4. See [#14270](https://github.com/DataDog/integrations-core/pull/14270).
+* [Fixed] Fix duplicate events bug. See [#14020](https://github.com/DataDog/integrations-core/pull/14020).
+* [Changed] Revamp Kafka consumer check. See [#13918](https://github.com/DataDog/integrations-core/pull/13918).
```

Comment:
Is this suppose to be here?

Reply:
Yeah, we added a new dependency and removed some old dependencies: https://github.com/DataDog/integrations-core/pull/13918/files#diff-1b02d4258aa29155f48813d2eb55fc87f62abbded6d8a9fa9e3b8f08365d771a

Reply:
I think he means since this entry doesn't mean anything for the base package we should either remove it or change it to talk about the dependency

Reply:
Yeah essentially. It felt out of place to me. That's all

Reply:
👍 , I'll rename this entry to be more related for the base check.